### PR TITLE
Add a bio file for GitHub Teacher

### PR DESCRIPTION
**Pull request by githubteacher**
This Pull Request adds a bio file for @githubteacher as requested in #2.

@githubstudent I would :heart: some :eyes: and :thought_balloon: on this when you have some :watch:.

### CODE REVIEW

**Comment by githubteacher** [comment]
Any others who would like to leave feedback are welcome to!  I :heart: your input!

File: docs/githubteacher.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# GitHub Teacher
+
+- **Name:** GitHub Teacher
+- **Occupation:** Teacher
+- **Location:** San Francisco
```
Comment (githubteacher) [inline docs/githubteacher.md]: Maybe we should be more specific here.  There are lots of San Franciscos in the :earth_americas:.

**Comment by githubteacher** [comment]
:+1: 
:ship: it!
:shipit:

**Comment by ghost** [comment]
:+1: Nothing dangerous here, all aspects covered

**Pull request merged**